Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs

```diff
@@ -128,6 +143,20 @@ public void TestTimeParseLuis()
             BasicTest("I'll be back noon", "T12");
             BasicTest("I'll be back 11ish", "T11");
             BasicTest("I'll be back 11-ish", "T11");
+
+            BasicTest("midnight", "T00");
+            BasicTest("mid-night", "T00");
+            BasicTest("mid night", "T00");
+            BasicTest("midmorning", "T10");
+            BasicTest("mid-morning", "T10");
+            BasicTest("mid morning", "T10");
+            BasicTest("midafternoon", "T14");
```

Comment:
Can we add a comment to the code mentioning exactly where on the spec is the definition of afternoon?
It's very counter intuitive that midafternoon resolves to T14.